azext_iot/iothub/_help.py

```diff
@@ -316,6 +316,53 @@ def load_iothub_help():
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
 
+    helps["iot edge devices"] = """
+        type: group
+        short-summary: Commands to manage IoT Edge devices.
+    """
+
+    helps["iot edge devices create"] = """
+        type: command
+        short-summary: Create multiple edge devices in an IoT Hub.
+        long-summary: |
+          This operation accepts inline device arguments or an edge devices configuration file in YAML or JSON format.
+          Inline command args (like '--device-auth') will take precedence and override configuration file properties if they are provided.
+          A sample configuration file can be found here: https://github.com/Azure/azure-iot-cli-extension/tree/dev/docs/samples/sample_devices_config.yaml
```

Comment:
nit/side thought, should we use an aka.ms link for the sample? 

Unrelated to this PR I want to use an aka.ms link initially pointing to our wiki dedicated page for inline json help for json heavy commands.

Reply:
Yeah, this absolutely should be an aka.ms link - not only can we then update URLs if anything changes (branch/folder structure, filename) without updating help docs, but it's also shorter.